packages/amplify-console-hosting/utils/amplify-console-utils.js

```diff
@@ -0,0 +1,112 @@
+const { put } = require('request-promise');
+const fs = require('fs-extra');
+const ora = require('ora');
+
+const DEPLOY_ARTIFACTS_MESSAGE =
+  'Deploying build artifacts to the Amplify Console..';
+const DEPLOY_COMPLETE_MESSAGE = 'Deployment complete!';
+const DEPLOY_FAILURE_MESSAGE = 'Deploy failed!';
```

Comment:
Deployment failed: <Reason>! Please report an issue on the Amplify Console GitHub issue tracker at https://github.com/aws-amplify/amplify-console/issues.